caffe2/mobile/contrib/opencl/operators/conv_op.cc

```diff
@@ -90,7 +90,9 @@ bool CLConvOp<T, Activation>::RunOnDevice() {
   LOG(ERROR) << "[C2DEBUG] group: " << group_ << " depthwise: " << depthwise << " grouped: " << grouped;
   CAFFE_ENFORCE(C % group_ == 0);
   CAFFE_ENFORCE(M % group_ == 0);
-  CAFFE_ENFORCE(filter_->dim32(1) == C / group_);
+  if (!depthwise) {
+    CAFFE_ENFORCE(filter_->dim32(1) == C / group_);
```

Comment:
This condition should be met regardless whether it is depthwise or not right? Or is this change just for debugging purpose?

Reply:
Yes, I'll revert both.